controllers/v1/surveysController.js

```diff
@@ -871,4 +868,77 @@ module.exports = class Surveys extends Abstract {
 
 }
 
+
+/**
+    * @api {get} /surveys/getStartedSurveys/{{programId}}
+    * @apiVersion 1.0.0
+    * @apiName Get survey and observation documents in program
+    * @apiGroup Internal API
+    * @apiHeader {String} X-authenticated-user-token Authenticity token
+    * @apiParam {String} programId.
+    * @apiSampleRequest /surveys/getStartedSurveys/63a42786c0b15a0009f0505e
+    * @apiUse successBody
+    * @apiUse errorBody
+    * @apiParamExample {json} Response:
+    {
+   "success":true,
+   "message":"Surveys fetched successfully",
+   "status":200,
+   "result":[
+         {
+            "_id":"64639270a0efdd0008575952",
+            "solutionId":"63690d9e743f760009155f6b",
+            "solutionExternalId":"2546ecb8-407f-11ec-8473-7fe753029532-1667829150737",
+            "programId":"6319a4d53c40dd000978dacb",
+            "programExternalId":"PGM-FD558-testing_program-5.0"
+         },
+         {
+            "_id":"64639279a0efdd000857595a",
+            "solutionId":"6369011a743f760009155e9b",
+            "solutionExternalId":"2546ecb8-407f-11ec-8473-7fe753029532-1667825946531",
+            "programId":"6319a4d53c40dd000978dacb",
+            "programExternalId":"PGM-FD558-testing_program-5.0"
+         },
+      ],
+}
+    */
+    /**
+   * Get survey documents for program.
+   * @method
+   * @name getStartedSurveys
+   * @param {Object} req -request Data.
+   * @param {String} req.params._id - programId.
+   * @returns {JSON} 
+   */
+
+    async getStartedSurveys(req,res){
```

Comment:
@ankitpws Let's move this new API under the "users" controller with the name "surveys"

Reply:
Created under users controller